libqfieldsync/offline_converter.py

```diff
@@ -272,9 +285,14 @@ def _convert(self, project: QgsProject) -> None:
             elif layer_action == SyncAction.REMOVE:
                 project.removeMapLayer(layer)
 
-        export_project_filename = self.export_folder.joinpath(
-            f"{self.original_filename.stem}_qfield.qgs"
-        )
+        if self.export_type == ExportType.Cable and self._export_filename:
+            export_project_filename = self.export_folder.joinpath(
+                f"{Path(self.backup_filename).stem}_qfield.qgs"
+            )
+        else:
+            export_project_filename = self.export_folder.joinpath(
+                f"{self.original_filename.stem}_qfield.qgs"
+            )
```

Comment:
```suggestion
        export_project_filename = self._export_filename
```

I guess in QFieldSync you need to make sure this file does not exists (yet) and ask for confirmation to overwrite it.